### DISCUSSION TITLE
how does everything works?

### DISCUSSION THREAD

**Discussion by MatteoPaget** [Lost-MSth/Arcaea-server#39] [Q&A]
i've been looking at this github for 30 minutes, but i still can't figure out how to use the server, any information about that?

**Comment by D-06** [comment]
Have you ever used python before

**Reply by MatteoPaget** [reply]
Yes, i'm currently learning it

**Reply by D-06** [reply]
So do you know how to use pip

**Reply by MatteoPaget** [reply]
Not really

**Reply by D-06** [reply]
Pip is the package installer for python it use for installing python package such as wheel,pandas,...

**Reply by D-06** [reply]
In this case you only need to install flask and cryptography
```
pip install flask cryptography
```
